src/Advection/centered_second_order.jl

```diff
@@ -22,6 +22,6 @@ const C2 = CenteredSecondOrder
 
 # Calculate the flux of a tracer quantity c through the faces of a cell.
 # In this case, the fluxes are given by u*Ax*T̅ˣ, v*Ay*T̅ʸ, and w*Az*T̅ᶻ.
-@inline advective_tracer_flux_x(i, j, k, grid, ::C2, U, c) = Ax_ψᵃᵃᶠ(i, j, k, grid, U) * ℑxᶠᵃᵃ(i, j, k, grid, c)
-@inline advective_tracer_flux_y(i, j, k, grid, ::C2, V, c) = Ay_ψᵃᵃᶠ(i, j, k, grid, V) * ℑyᵃᶠᵃ(i, j, k, grid, c)
-@inline advective_tracer_flux_z(i, j, k, grid, ::C2, W, c) = Az_ψᵃᵃᵃ(i, j, k, grid, W) * ℑzᵃᵃᶠ(i, j, k, grid, c)
+@inline advective_tracer_flux_x(i, j, k, grid, ::C2, U, c) = Ax_uᶠᶜᶜ(i, j, k, grid, U) * ℑxᶠᵃᵃ(i, j, k, grid, c)
```

Comment:
Similarly these new functions seem to be at different positions and don't see a problem but just wanted to check if the old code was part of the bug